src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -147,6 +204,7 @@ class DSWxS1Executor(DSWxS1PreProcessorMixin, DSWxS1PostProcessorMixin, PgeExecu
 
     NAME = "DSWx"
     """Short name for the DSWx-S1 PGE"""
+    # TODO do we not want to differentiate between S1 and HLS
```

Comment:
Not sure what this TODO is in reference to, can it be removed? Also, `NAME` should be set to `DSWx-S1`.